src/Saritasa.Tools.SourceGenerator/Saritasa.Tools.SourceGenerator.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
Since source generators can work's only with netstandard2.0 - many language features is locked, so make sure.